test/cloudant.test.js

```diff
@@ -113,7 +143,7 @@ describe('cloudant connector', function() {
       });
   });
 
-  describe('updateAll and updateAttributes', function() {
+  describeskip('updateAll and updateAttributes', function() {
```

Comment:
@raymondfeng currently all test cases are skipped on CI, but will run when testing on local. So does this one, I name it as describeskip but it will still run in local test.

We have an issue opened to fix all test cases but haven't got a chance to do it, and we are planning to clean up tests together when refactor the code to support couchdb.

I try to enable test cases related in this pr on CI while still keep others skipped.